### PR TITLE
Login for all Users with tests

### PR DESCRIPTION
**Pull request by JayMunk**
Login par controller -> service -> repository
Creation de test

### CODE REVIEW

File: src/main/java/com/group1/stagesWs/service/AppService.java [hunk 1]
```diff
@@ -26,8 +26,23 @@
     private SuperviseurRepository superviseurRepository;
 
 
-
-    public Etudiant addEtudiant(Etudiant etudiant){
+    public Etudiant addEtudiant(Etudiant etudiant) {
         return etudiantRepository.save(etudiant);
     }
+
+    public User login(String email, String pwd) {
```
Comment (ca-n) [inline src/main/java/com/group1/stagesWs/service/AppService.java:33]: Clean code: Too many if statements

File: src/test/java/com/group1/stagesWs/StageswsApplicationTests.java [hunk 2]
```diff
@@ -1,13 +1,98 @@
...
+        Superviseur superviseur1 = new Superviseur("Johny", "Cash", "cashJ@email.ca", "money123", "995-897-4562", "Informatique", "AI");
+        superviseurRepository.save(superviseur1);
+    }
+
     @Test
     void contextLoads() {
     }
 
+    @Test
+    void testFindEtudiant() {
```
Comment (ca-n) [inline src/test/java/com/group1/stagesWs/StageswsApplicationTests.java:59]: Pas necessaire de tester le Repository, spring le teste pour nous. Testez juste la Service

**Review by ca-n — CHANGES_REQUESTED** [review]
AppController should be renamed to UserController, as we will have different controllers for User, Notifications, etc.

File: src/main/java/com/group1/stagesWs/service/AppService.java (hunk 1 above)
Comment (JayMunk) [reply src/main/java/com/group1/stagesWs/service/AppService.java:33]: Did not find anything simpler than this

**Review by ca-n — APPROVED** [review]
Looks good :+1:

**Pull request merged**